master/internal/api_experiment.go

```diff
@@ -43,3 +53,104 @@ func (a *apiServer) GetExperiments(
 	a.sort(resp.Experiments, req.OrderBy, req.SortBy, apiv1.GetExperimentsRequest_SORT_BY_ID)
 	return resp, a.paginate(&resp.Pagination, &resp.Experiments, req.Offset, req.Limit)
 }
+
+func (a *apiServer) ActivateExperiment(
+	ctx context.Context, req *apiv1.ActivateExperimentRequest) (
+	resp *apiv1.ActivateExperimentResponse, err error) {
+	var rawExp []byte
+	var exp model.Experiment
+	if rawExp, err = a.m.db.ExperimentRaw(int(req.Id)); err != nil {
```

Comment:
blocking: All this logic should go into a simple query that fetches the ID only. This query is probably pretty expensive.